inc/templates/namespace.php

```diff
@@ -641,3 +643,88 @@ function inject_favicon( array $children, array $config, string $name ): array {
 		Components\html_to_components( get_favicon_markup(), [ 'link', 'meta' ] )
 	);
 }
+
+/**
+ * Inject the custom css styles into the <head> component.
+ *
+ * @param array  $children Children for this component.
+ * @param array  $config   Config for this component.
+ * @param string $name     Name of this component.
+ * @return array
+ */
+function inject_custom_css( array $children, array $config, string $name ): array {
```

Comment:
I think it might be cleaner to declare these functions in the `irving/head` component file, since it's a filter specifically for that component, or even to add an action inside the `children_callback` that these could hook to.

Reply:
Do you think the `add_action` calls should still be in namespace? Or somewhere else?

I'm not against adding a component-specific filter, but based on convos we've had before, I thought we were trying to avoid that pattern in favor of doing it via the `class-component.php`.